web-console/monitoring/alertmanager-config/alertmanager-config.md

```diff
@@ -0,0 +1,34 @@
+# Alertmanager Configuration
+
+![alerts list view](img/alerts-list.png)
+- To consolidate all admin-level alerting functions together, `Alerts` and `Silences` pages could be combined onto the same page as tabs with the `Configuration` and `YAMl` pages
+
+![alerting configuation tab](img/config-empty.png)
+- The `Configuration` tab would show some basic grouping information from the default route section of the YAML and a list of receivers for the cluster.
+
+![configuration tab with create menu open showing receiver types](img/config-dropdown.png)
+- Clicking the “create” dropdown would enable users to create a receiver based on an integration type.
+- We could provide the entire list of integration types here, or just a subset as time allows.
+- Users would only be able to create receivers with a single type of integration. More complex receivers could be created from the `YAML` tab.
+
+## Creating the default receiver
+![create receiver page](img/create-default-receiver.png)
+- The create receiver form would expose basic required information for the given integration type.
+- If there is no information in the `global` section of the config YAML for a given receiver type, possible global fields are added to the form with the option to write them to the `global` secton. If the "Save as default..." box is not checked, they are simply added to the receiver itself.
+- Optional fields appear if the user clicks the `Show advanced configuration` button.
+- If no receivers have been created, no other fields are available as this receiver must become the default receiver for alerts.
+
+## Creating subsequent receivers
+![create reciever page with routing labels section](img/create-receiver-long.png)
```

Comment:
The "Save as..." help tooltip gets a little technical here, which is fine, but I wonder if something like "These settings will be applied to future email receivers by default." would suffice?